### DISCUSSION TITLE
[1.2.0] Phandelver and Below Changelog

### DISCUSSION THREAD

**Discussion by mattexdee** [foundryvtt/foundryvtt-premium-content#594] [Updates]
## Feature Enhancements
- In V12 the Doppelganger Disguise Switcher now includes an animation argument which provides a fancy morphing effect during the transition.
- All Art Handouts and Atlas journals now have a default ownership of Observer with internal pages ownership set to None. This allows players to see the journals but not their contents until the Dungeon Master shows them and optionally grants them observer permission to the page. [[#521]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/521>)
- Configured PBSO as a Book Source.
- Added a warning to the tile rotation macros that will run if the module is not activated that notifies users the module must be active for them to work correctly.
- Improved the lighting in Zorzula's Rest to not be overwhelmingly bright for tokens with darkvision.
- Improved the walling in Indigo Sanctum to allow tokens to see Ruxithid when they enter the chamber.
- Enabled Fog Exploration in Zorzula's Rest and Indigo Sanctum.

## Bug Fixes
- Standardized the source details for all Actors. [[#483]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/483>)
- Added the Psychic Rend attack to the Aberrant Zealot. [[#505]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/505>)
- Fixed the link to the Shambling Mound in the Actor Index. [[#510]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/510>)
- Fixed an issue with the Doppelganger Disguise Switcher Macro not working in V12. [[#579]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/579>)
- Fixed the reference to the surprised condition in the Umber Hulk's Petrifying Gaze feature. [[#544]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/544>)
- Moved the Shelter of the Faithful feature to the Background Features folder. [[#531]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/531>)
- Tool and skill proficiencies provided by backgrounds can now be replaced. [[#530]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/530>)
- Added a missing check and removed duplicate damage type in the Goblin Trail. [[#523]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/523>)
- The Brazier of Green Flame no longer allows for enchanting magical items. [[#516]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/516>)
- Updated the detail links in "What the Redbrands Know" to reference R4 for the skeletal guard room and R5 for the prisoner room. [[#589]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/589>)
- Zeond is now correctly configured in the Scene as invisible and Iarno "Glasstaff" Albrek's name now appears as "Glasstaff". [[#590]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/590>)
- Added languages to Harbin Wester and Sister Garaele.
- Fixed a number of issues. [[#515]](<https://github.com/foundryvtt/foundryvtt-premium-content/issues/515>)
    - Standardized the source data for all Items.
    - Improved the configuration of the Staff of Defense by using the new D&D 5E Item enricher and configured charges to be used by the provided spells.
    - Set sub-type values for any loot items that had fitting sub-types.
    - Added rarities where necessary.
    - Added weight to Sildar's Longsword.
    - Removed span styles from Dragonguard description.
    - Added magic properties to relevant items.
    - Added damage type to Bead of Force.
    - Added Active Effect to Periapt of Health.
    - Configured Ring of the Orator and added an Active Effect for tracking its effect.
    - Configured recovery formulas where necessary.

**Comment by daniels070** [comment]
Nice

**Comment by gbradburn** [comment]
Are there plans to update this for the new 2024 rules/content?
